sop-website/docs/Data-Upload-Update/Data-Uploads.mdx

```diff
@@ -0,0 +1,418 @@
+          ├── 10001001_20220401_113120_3600_0002.dat
+          ...
+```
+
+This structure shall be maintained for all submissions.  All additions, deletions, and modifications shall be made in this structure.  
+
+A relaxation of the folder structure is that OMOP, images, and waveforms files may be stored in separate file systems.  For example, images may be stored in the AWS cloud, while waveforms may be stored in an on-prem Linux file system.  Each file system shall maintain the same organization, for example:
+
+In AWS S3 bucket:
+```
+/10001001
+  ├── Images
+      ├── 1.2.840.113619.2.176.3596.6358730.30068.1592383800.1.1
+          ├── 1.2.840.113619.2.176.3596.6358730.30068.1592383800.2.1
+              ├── 10001001_20220401_113120_CT_01_0.dcm
+              ├── 10001001_20220401_113120_CT_01_1.dcm
+              ├── 10001001_20220401_113120_CT_01_2.dcm
+              ...
+```
+
+
+## Central Staging File Organization
+This virtual environment is hosted in an Azure platform. The containers structure is setup such that each DGS has its own container (e.g. mgh).  The container name for each DGS is used in the `config.toml` files as `{DGS_container}` in the [Configuration](#configuration) section.
+
+![Central Data Organization](https://github.com/chorus-ai/data_acq_SOP/assets/2847495/d7d1d67a-2695-4b2d-92aa-2285b608a09d)
+
+### Central Staging Folder Structure (INFORMATIONAL)
+The submitted data will be organized using the same folder structure as the DGS local file store, except that data for each submission will placed in a top level folder named with a timestamp of the format `YYYYMMDDhhmmss`.  The folder structure is shown here for context.  The timestamped folder will be created automatically by the `chorus-upload` tool.  
+
+![Central Data Organization](https://github.com/chorus-ai/Chorus_SOP/blob/review-data-upload-update/sop-website/docs/Data-Upload-Update/Central_Structure.png?raw=true)
+
+In the diagram above, the gray directories were added, blue folder has modifications, and orange folder has been deleted.
+
+
+# Data Preparation and Upload Process
+
+## For Azure Data Share Users
+
+1. The DGS local data shall be organized according to the [File Organization](#file-organization) section.
+2. The Azure Data Share shall be configured to copy data into a "staging" folder in the DGS container.
+3. (INFORMATIONAL) The staging data will be scanned to create/update a journal database (by `chorus-upload`) and copied to the a dated directory monthly.
+
+## For Current AZ CLI Users
+
+The `chorus-upload` tool is required for sites that are not using Azure Data Share.  
+
+The code is available in [chorus-extract-upload](https://github.com/chorus-ai/chorus-extract-upload) repository.
+
+### Installation
+
+The `chorus-upload` tool is a Python package.  The package can be installed using pip.  The package requires Python 3.7 or later.  A virtual environment (venv or conda) is strongly recommended.
```

Comment:
Most the content in this section seems to be in the Github README already. It'd probably be more sustainable to just maintain one copy in the repo and refer to it from here instead of keeping two copies in sync.

Reply:
moved.